src/controllers/comments_controller.ts

```diff
@@ -53,7 +54,7 @@ const updateCommentById = async (req, res) => {
         .json({ error: "Content and sender are required." });
     }
 
-    const updatedComment = await Comment.findByIdAndUpdate(
+    const updatedComment: CommentDocument | null = await Comment.findByIdAndUpdate(
```

Comment:
Are you sure it returns null and not undefined?

Reply:
according to typescript suggestion